package.json

```diff
@@ -28,6 +28,7 @@
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
     "react-redux": "^7.2.0",
+    "react-router-dom": "^5.1.2",
```

Comment:
-you don't need line 31 anymore right because you aren't using react router?    
-Do you even need 28-30?  Are you using redux?  
-You are using the react cdn's as script tags so you shouldn't need react and react-dom in package.json should you?